module-types/Datacom/PSU-125-DC.yaml

```diff
@@ -0,0 +1,10 @@
+---
+manufacturer: Datacom
+model: PSU 125 DC
+part_number: 800.5188.xx
+comments: 125W power supply with -48VDC input compatible with OLT DM4615. Allows operation as a redundant and hot-swappable source.
+power-ports:
+  - name: PSU
```

Comment:
Please update this name to use the `{module}` definition

Reply:
sorry, but I don't understand how it should be